configure.ac

```diff
@@ -16,9 +16,11 @@ AM_PROG_CC_C_O
 AC_DISABLE_STATIC
 AC_PROG_LIBTOOL
 
-dnl i18n
-IT_PROG_INTLTOOL([0.35.0])
-GP_I18N
```

Comment:
If we don't call `GP_I18N` here anymore, couldn't we remove `build/i18n.m4` completely?

If so, we probably need to declare `LOCALEDIR` here or still use `GP_I18N`.